web/src/components/vis/ScorePlotTile.vue

```diff
@@ -54,7 +84,11 @@ vis-tile(v-if="plot", title="PCA Score Plot", :loading="plot.loading")
   score-plot(
       :width="width",
       :height="height",
-      :raw-points="plot.data",
+      :pc-coords="pcCoords",
+      :row-labels="rowLabels",
+      :group-labels="groupLabels",
+      :eigenvalues="eigenvalues",
+      :columns="columns",
       :dataset="dataset",
```

Comment:
Looks like dataset was removed from the score-plot props.

Reply:
See 2b7a292eddc114ddcbc25e20faa21938ff0c2bb8.